### ISSUE TITLE
Maximum & minimum does not work at parameters

### ISSUE THREAD

**Issue by lovesuper** [spec-first/connexion#882]
### Description

```
limit:
  description: The number of items to return
  in: query
  name: limit
  required: true
  schema:
    type: integer
    minimum: 1
    maximum: 1000

```
### Expected behaviour

Numbers above and below 1 and 1000 must to decline by server

### Actual behaviour

They are being accepted

### Steps to reproduce

Just try to validate integer by `maximum` and `minimum` at parameters

### Additional info:

Output of the commands:

- `Python 3.7.2`
- `Version: 2018.0.dev1`

**Comment by dtkav** [comment]
Hey @lovesuper - we use https://github.com/p1c2u/openapi-spec-validator in order to validate incoming data against the spec.
Can you file a ticket there?

**Comment by dtkav** [comment]
hey @lovesuper  - I misread you ticket - This is not related to `openapi-spec-validator` and is actually related to `jsonschema`. Is `maximum` in the Draft4 jsonschema spec?

**Comment by hamid914** [comment]
fixed in v2.2.0 a0ac0fd

**Comment by RobbeSneyders** [comment]
Fixed
